File: backend/src/main/java/net/dnadas/training_portal/dto/group/project/questionnaire/QuestionnaireSubmissionResponseEditorDto.java
package net.dnadas.training_portal.dto.group.project.questionnaire;

import net.dnadas.training_portal.model.group.project.questionnaire.QuestionnaireStatus;

public record QuestionnaireSubmissionResponseEditorDto(
  Long id, String name, String description, Integer receivedPoints, Integer maxPoints,
  String createdAt, QuestionnaireStatus status) {
}

File: frontend/src/questionnaires/dto/QuestionnaireSubmissionResponseEditorDto.ts
import {QuestionnaireStatus} from "./QuestionnaireStatus.ts";

export interface QuestionnaireSubmissionResponseEditorDto {
  id: number;
  name: string;
  description: string;
  receivedPoints: number;
  maxPoints: number;
  createdAt: string;
  status: QuestionnaireStatus;
}
